src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -472,6 +472,7 @@
 
         :else
         {:status 200
+         :headers {"Content-Type" "application/octet-stream"}
```

Comment:
> I was musing about the slightly wider case - if a given response doesn't have a content-type at all should we add one?

Oh, you mean like at a higher level in the middleware stack we might default in a Content-Type if none had been set in the response map at the lower-level?  I tend to think it's best to avoid a higher-level default since you may not have as much context there to know what the body represents.

If we do `text/plain` for each of the error clauses and `application/octet-stream` for success for this endpoint, I think we have all of the `cond` clauses covered for this endpoint anyway.